src/sentry/api/serializers/models/exporteddata.py

```diff
@@ -1,25 +1,31 @@
-from sentry.api.serializers import Serializer, register, serialize
+from sentry.api.serializers import Serializer, register
 from sentry.data_export.base import ExportQueryType
 from sentry.data_export.models import ExportedData
-from sentry.models import User
+from sentry.services.hybrid_cloud.user import user_service
 
 
 @register(ExportedData)
 class ExportedDataSerializer(Serializer):
     def get_attrs(self, item_list, user, **kwargs):
         attrs = {}
-        users = User.objects.filter(id__in={item.user_id for item in item_list})
-        user_lookup = {user.id: user for user in users}
+        serialized_users = {
+            u["id"]: u
+            for u in user_service.serialize_many(
+                filter=dict(user_ids=[item.user_id for item in item_list])
+            )
+        }
```

Comment:
We seem to do this a bunch. Should the `user_service` expose a `serialize_many_map` method?

Reply:
In, general, I think the `serialize_many` should be returning a dictionary, so I agree.  I had been thinking about making a larger refactor for this purpose.

I'd still prefer to keep the service interfaces thing (one way of doing things, not 3), but in this case returning by dictionary is actually preferable to the list, so I may replace it that way.

Reply:
I'll make some follow up work to consider refactoring all serialize_many call sites into this convention.